### PR TITLE
Added Chapter 3.4 - Posting comments and removing friends

### PR DESCRIPTION
**Pull request by DentFuse**

### CODE REVIEW

File: Chapter 3 - User Interaction/Chapter 3.4 - Commenting on Profile & Removing Friends/project4.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  } else if (message === "!comment") {
+    client.chatMessage(steamid,"Commenting on your profile!");
+    community.postUserComment(steamid, "My comment");
+  } else if (message === "!remove") {
+    client.chatMessage(steamid,"See you again later...")
+    client.removeFriend(steamid)
+  } else {
+    client.chatMessage(steamid,"I failed to understand you :/")
+  }
+});
```
Comment (andrewda) [inline Chapter 3 - User Interaction/Chapter 3.4 - Commenting on Profile & Removing Friends/project4.js]: Newline at EOF

**Review by andrewda — CHANGES_REQUESTED** [review]

File: Chapter 3 - User Interaction/Chapter 3.4 - Commenting on Profile & Removing Friends/project4.js (hunk 1 above)
Comment (DentFuse) [reply Chapter 3 - User Interaction/Chapter 3.4 - Commenting on Profile & Removing Friends/project4.js]: Added newline at EOF in 364f48e

**Comment by DentFuse** [comment]
@andrewda Hi, can you please merge this PR, I'd do it myself but I'm actually taking part in the hacktoberfest and so I don't want to have my own pull requests approved by me.

**Comment by xmwx38** [comment]
All done.  It has been merged!  @DentFuse

**Pull request merged**